### PR TITLE
Remove connector ops team

### PR DESCRIPTION
**Pull request by evantahler**
```[tasklist]
### Tasks
- [x] `airbyte` Repo: Codeowners, Airbyte CI automations (https://github.com/airbytehq/airbyte/pull/34867)
- [x] `airbyte-Platform-Internal` Repo: Codeowners, etc (https://github.com/airbytehq/airbyte-platform-internal/pull/11081)
- [x] `workflow-actions` Repo: https://github.com/airbytehq/workflow-actions/pull/86
- [x] Remove/Archive Slack Team
- [x] Remove Github Team
- [x] Remove Slack Group
```

### CODE REVIEW

File: .github/workflows/publish-command.yml [hunk 1]
```diff
@@ -45,4 +45,4 @@ jobs:
             The connector publication happens on merge to the master branch.<br>
             Please use /legacy-publish if you need to publish normalization images.<br>
             Please join the #connector-publish-updates slack channel to track ongoing publish pipelines.<br>
-            Please reach out to the @dev-connector-ops team if you need support in publishing a connector.
+            Please reach out to the @dev-extensibility team if you need support in publishing a connector.
```
Comment (alafanechere) [inline .github/workflows/publish-command.yml:48]:
```suggestion
            Please reach out to the @airbytehq/connector-extensibility team if you need support in publishing a connector.
```

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py [hunk 2]
```diff
@@ -133,5 +133,5 @@ def create_slack_message(self) -> str:
             assert self.report is not None, "Report should be set when state is successful"
             message += f"⏲️ Run duration: {format_duration(self.report.run_duration)}\n"
         if self.state is ContextState.FAILURE:
-            message += "\ncc. <!subteam^S0407GYHW4E>"  # @dev-connector-ops
+            message += "\ncc. <!subteam^dev-connector-extensibility>"
```
Comment (alafanechere) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py:136]: Does this work? I thought we had to figure the group id.

File: airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py
Comment (alafanechere) [inline airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/publish/context.py:1]: Please bump airbyte-ci version in `pyproject.toml` (+ changelog update in `README.md`) if you want this change to take affect following the merge of this PR

File: .github/workflows/publish-command.yml
Comment (alafanechere) [inline .github/workflows/publish-command.yml:1]: I think we can remove / disable this workflow as /publish is gone

**Review by alafanechere — CHANGES_REQUESTED** [review]

**Comment by evantahler** [comment]
FYI @natikgadzhi this PR tracks all the code changes I'm doing to wind down the connector ops team, if you ever want to change something.

File: airbyte-ci/connectors/pipelines/README.md
Comment (alafanechere) [inline airbyte-ci/connectors/pipelines/README.md:1]: Are you using a specific markdown formatter locally?

**Review by alafanechere — APPROVED** [review]
Thank you for the changes 🙏

File: airbyte-ci/connectors/pipelines/README.md
Comment (evantahler) [reply airbyte-ci/connectors/pipelines/README.md:1]: Just vscode 🤷

**Pull request merged**